pkgs/development/python-modules/gdown/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonApplication
+, fetchPypi
+, filelock
+, requests
+, tqdm
+, setuptools
+}:
+
+buildPythonApplication rec {
+  pname = "gdown";
+  version = "3.11.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1p023812hh7w7d08njjsfn0xzldl4m73yx8p243yb2q49ypjl6nz";
+  };
+
+  propagatedBuildInputs = [ filelock requests tqdm setuptools ];
+
+  doCheck = false;
+
```

Comment:
This disables the checkPhase, running the tests is already disabled by overriding the default checkPhase, so you're good
```suggestion
```